Cargo.lock

```diff
@@ -1,6 +1,6 @@
 [[package]]
 name = "aho-corasick"
-version = "0.6.4"
+version = "0.6.5"
```

Comment:
You may update only tox package by `cargo update -p tox`

Reply:
Does it make sense? Since everything is working fine with updated versions.